### PR TITLE
Bump Monitoring V1 images and resolve deprecation notices

### PR DESCRIPTION
**Pull request by aiyengar2**
Related Issue: https://github.com/rancher/rancher/issues/32839

Pending:
- [x] https://github.com/rancher/image-mirror/pull/130

### CODE REVIEW

**Review by brendarearden — APPROVED** [review]

File: charts/rancher-monitoring/v0.2.3/charts/operator-init/files/crd-alertmanager.yaml [hunk 1]
```diff
@@ -1,9 +1,8 @@
-# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/d8d7949edf839ffb7fae859992c05c894b373862/bundle.yaml
+# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.40.0/bundle.yaml
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    helm.sh/hook: crd-install
```
Comment (jiaqiluo) [inline charts/rancher-monitoring/v0.2.3/charts/operator-init/files/crd-alertmanager.yaml:6]: why is this removed? 
Is it removed by the upstream? 

-------
Update:

Please ignore this comment since it is not an issue. the charts/rancher-monitoring/v0.2.3/charts/operator-init has another job to handle the installation of CRDs properly: https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-monitoring/v0.2.3/charts/operator-init/templates/job-install-crds.yaml

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
@aiyengar2  Should we bump the version of the chart to 0.2.4?

File: charts/rancher-monitoring/v0.2.3/charts/operator-init/files/crd-alertmanager.yaml (hunk 1 above)
Comment (aiyengar2) [reply charts/rancher-monitoring/v0.2.3/charts/operator-init/files/crd-alertmanager.yaml:6]: yes it is removed by upstream and yes we effectively do the CRD (sub)chart install in Monitoring V1

**Comment by aiyengar2** [comment]
> @aiyengar2 Should we bump the version of the chart to 0.2.4?

@jiaqiluo 0.2.3 has not been released yet, so I don't think we need to bump.

https://github.com/rancher/system-charts/tree/release-v2.5/charts/rancher-monitoring

**Review by jiaqiluo — APPROVED** [review]
LGTM
